package.json

```diff
@@ -61,6 +61,7 @@
           "data-export.config.item.get",
           "data-export.job.collection.get",
           "data-export.job.item.get",
+          "data-export.job.item.post",
```

Comment:
I assume it should be added to `Export manager: Download and re-send files` rather than `Export manager: All`